app/static/js/controllers/notificationController.js

```diff
@@ -28,13 +28,13 @@ function ($scope, $rootScope, $filter, Cluster, GlobalStack) {
       var eventType = notification.eventType;
 
       if (successEvents.indexOf(eventType) > -1) {
-        handleStatusChange(notification, "has-success");
```

Comment:
We should call 'handleStatusChange' function to actual cluster's status be set and notification be added to the global list.